tests/nightly/nightly-kickoff.sh

```diff
@@ -36,46 +45,701 @@ tar xvzf $input.tar.gz -C bin/ --strip 1
+RenameLogs "5-1-Distributed-Switch"
 
 drone exec --trusted -e test="pybot tests/manual-test-cases/Group5-Functional-Tests/5-2-Cluster.robot" -E nightly_test_secrets.yml --yaml .drone.nightly.yml
 
-mv output.xml output_5-2-Cluster.xml
-mv log.html log_5-2-Cluster.html
-mv vic-machine.log vic-machine_5-2-Cluster.log
+if [ $? -eq 0 ]
+then
+ClusterStatus="Passed"
+else
+ClusterStatus="FAILED!"
+fi
+
+RenameLogs "5-2-Cluster"
 
 drone exec --trusted -e test="pybot tests/manual-test-cases/Group5-Functional-Tests/5-4-High-Availability.robot" -E nightly_test_secrets.yml --yaml .drone.nightly.yml
+if [ $? -eq 0 ]
+then
+HighAvailabilityStatus="Passed"
+else
+HighAvailabilityStatus="FAILED!"
+fi
 
-mv output.xml output_5-4-High-Availability.xml
-mv log.html log_5-4-High-Availability.html
-mv vic-machine.log vic-machine_5-4-High-Availability.log
+RenameLogs "5-4-High-Availability"
 
 drone exec --trusted -e test="pybot tests/manual-test-cases/Group5-Functional-Tests/5-5-Heterogenous-ESXi.robot" -E nightly_test_secrets.yml --yaml .drone.nightly.yml
+if [ $? -eq 0 ]
+then
+HeterogenousStatus="Passed"
+else
+HeterogenousStatus="FAILED!"
+fi
 
-mv output.xml output_5-5-Heterogenous-ESXi.xml
-mv log.html log_5-5-Heterogenous-ESXi.html
-mv vic-machine.log vic-machine_5-5-Heterogenous-ESXi.log
+RenameLogs "5-5-Heterogenous-ESXi"
 
 drone exec --trusted -e test="pybot tests/manual-test-cases/Group5-Functional-Tests/5-6-VSAN.robot" -E nightly_test_secrets.yml --yaml .drone.nightly.yml
+if [ $? -eq 0 ]
+then
+VSANStatus="Passed"
+else
+VSANStatus="FAILED!"
+fi
 
-mv output.xml output_5-6-VSAN.xml
-mv log.html log_5-6-VSAN.html
-mv vic-machine.log vic-machine_5-6-VSAN.log
+RenameLogs "5-6-VSAN"
 
 drone exec --trusted -e test="pybot tests/manual-test-cases/Group5-Functional-Tests/5-7-NSX.robot" -E nightly_test_secrets.yml --yaml .drone.nightly.yml
+if [ $? -eq 0 ]
+then
+NSXStatus="Passed"
+else
+NSXStatus="FAILED!"
+fi
 
-mv output.xml output_5-7-NSX.xml
-mv log.html log_5-7-NSX.html
-mv vic-machine.log vic-machine_5-7-NSX.log
+RenameLogs "5-7-NSX"
 
 drone exec --trusted -e test="pybot tests/manual-test-cases/Group5-Functional-Tests/5-8-DRS.robot" -E nightly_test_secrets.yml --yaml .drone.nightly.yml
+if [ $? -eq 0 ]
+then
+DRSStatus="Passed"
+else
+DRSStatus="FAILED!"
+fi
 
-mv output.xml output_5-8-DRS.xml
-mv log.html log_5-8-DRS.html
-mv vic-machine.log vic-machine_5-8-DRS.log
+RenameLogs "5-8-DRS"
+
+if [[ $DistributedSwitchStatus = "Passed" && $ClusterStatus = "Passed" && $HighAvailabilityStatus = "Passed" && $HeterogenousStatus = "Passed" && $VSANStatus = "Passed" $NSXStatus = "Passed" $DRSStatus = "Passed" ]]
+then
+buildStatus=0
+else
+buildStatus=1
+fi
+echo "Global Nightly Test Status $buildStatus"
 
 drone exec --trusted -e test="sh tests/nightly/upload-logs.sh $input" -E nightly_test_secrets.yml --yaml .drone.nightly.yml
+
+rm nightly_mail.html
+
+if [ $buildStatus -eq 0 ]
+then
+echo "Success"
+cat <<EOT >> nightly_mail.html
+To: rashok@vmware.com
```

Comment:
This is a sample email from the html template, and by the way using SendMail instead of drone for email notification. Its complicated with running multiple drone execs to gather test statuses. SendMail is good.